### PR TITLE
Remove ts from conf

### PR DESCRIPTION
**Pull request by 0xDiBa**
Explicit truststore configuration.
Only path allowed for certain truststores.

### CODE REVIEW

File: certificate/analyser.go [hunk 1]
```diff
@@ -19,17 +19,51 @@ import (
...
+
+			path = ts.MicrosoftTS
+
+		case apple_TS_name:
+			path = ts.AppleTS
+
+		case android_TS_name:
+			path = ts.AndroidTS
+
+		default:
```
Comment (jvehent) [inline certificate/analyser.go]: stylistic: please remove the newlines, it's just as readable without them ;)

File: certificate/analyser.go [hunk 2]
```diff
@@ -19,17 +19,51 @@ import (
...
+
+			log.WithFields(logrus.Fields{
+				"tsname": name,
+			}).Warning("Wrong slice name entry. This should not be there")
```
Comment (jvehent) [inline certificate/analyser.go]: need a more explicit error message, such as
`Truststore %q is not a valid name", name`

**Comment by jvehent** [comment]
I like it. there's just a couple nits, and it's good to merge!

**Comment by 0xDiBa** [comment]
Need to work on my styling skills ;)
Merging...

**Comment by 0xDiBa** [comment]
Did not realise that.
Add work to my common understanding skill.

Coming right up on my next commit ;)

**Pull request merged**